### ISSUE TITLE
CLI interpreter working on key words, will work for same line, due to method will not work for multi-lines

### ISSUE THREAD

**Issue by williamsuttondev** [williamsuttondev/commentcommit#4]
An example could be as follows in current state:
```
### @issue: hello world!
```
Would be interpreted correctly but:
```
### @issue: hello world!
           same command contents
```
Would not.

Need to deal with this in the way we're dealing with lines in the iterator inside of the commands function

**Comment by williamsuttondev** [comment]
Some progress being made on this bug: c4592b38879cf6280a722c9b9b45bb45c24dfb7c

**Comment by williamsuttondev** [comment]
Fixed: 25ad81f065a0dc18b9a6cd9093160a6477d3ab02
